docs/conf.py

```diff
@@ -13,6 +13,8 @@
 import os
 import sys
 
+import torch
```

Comment:
[Q] why are we importing this?

Reply:
see below, we're reading the members of nn.Module to exclude them from child classes